src/features/workspace/components/__tests__/workspace-muxing-model.test.tsx

```diff
@@ -0,0 +1,106 @@
+import { render } from "@/lib/test-utils";
+import { screen, waitFor } from "@testing-library/react";
+import { WorkspaceMuxingModel } from "../workspace-muxing-model";
+import userEvent from "@testing-library/user-event";
+
+test("render model overrides", async () => {
```

Comment:
```suggestion
test("renders model overrides", async () => {
```

but actually it seems like this test does a lot more than that